### PR TITLE
Update qiskit-terra to qiskit in README.md

### PR DESCRIPTION
**Pull request by 1ucian0**
The badges in the readme are pointing to qiskit-terra

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Qiskit
 [![License](https://img.shields.io/github/license/Qiskit/qiskit-terra.svg?)](https://opensource.org/licenses/Apache-2.0) <!--- long-description-skip-begin -->
```
Comment (jakelishman) [inline README.md]:
```suggestion
[![License](https://img.shields.io/github/license/Qiskit/qiskit.svg?)](https://opensource.org/licenses/Apache-2.0) <!--- long-description-skip-begin -->
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-[![Downloads](https://img.shields.io/pypi/dm/qiskit-terra.svg)](https://pypi.org/project/qiskit-terra/)
-[![Coverage Status](https://coveralls.io/repos/github/Qiskit/qiskit-terra/badge.svg?branch=main)](https://coveralls.io/github/Qiskit/qiskit-terra?branch=main)
+[![Release](https://img.shields.io/github/release/Qiskit/qiskit.svg)](https://github.com/Qiskit/qiskit/releases)
+[![Downloads](https://img.shields.io/pypi/dm/qiskit.svg)](https://pypi.org/project/qiskit/)
+[![Coverage Status](https://coveralls.io/repos/github/Qiskit/qiskit/badge.svg?branch=main)](https://coveralls.io/github/Qiskit/qiskit?branch=main)
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/qiskit)
 [![Minimum rustc 1.70](https://img.shields.io/badge/rustc-1.70+-blue.svg)](https://rust-lang.github.io/rfcs/2495-min-rust-version.html)
-[![Downloads](https://static.pepy.tech/badge/qiskit-terra)](https://pepy.tech/project/qiskit-terra)<!--- long-description-skip-end -->
+[![Downloads](https://static.pepy.tech/badge/qiskit)](https://pepy.tech/project/qiskit)<!--- long-description-skip-end -->
 [![DOI](https://zenodo.org/badge/161550823.svg)](https://zenodo.org/badge/latestdoi/161550823)
```
Comment (jakelishman) [inline README.md]: I imagine this Zenodo DOI points to the wrong place now as well?

**Review by jakelishman — COMMENTED** [review]
Thanks for spotting it!

**Review by Eric-Arellano — DISMISSED** [review]

**Review by jakelishman — APPROVED** [review]
Thanks Luciano.  I updated the Zenodo DOI to point to https://doi.org/10.5281/zenodo.2583252, which is the tracker for Qiskit/qiskit (a name that no longer contains Terra), rather than the old one which was the tracker for Qiskit/qiskit-metapackage.

**Pull request merged**